diff --git a/data/io.github.fabrialberio.pinapp.appdata.xml.in b/data/io.github.fabrialberio.pinapp.appdata.xml.in
index f5e5850..81021b5 100644
--- a/data/io.github.fabrialberio.pinapp.appdata.xml.in
+++ b/data/io.github.fabrialberio.pinapp.appdata.xml.in
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
 	<id>io.github.fabrialberio.pinapp</id>
-	<launchable type="desktop-id">io.github.fabrialberio.pinapp.desktop</launchable>	<name translatable='no'>PinApp</name>
+	<launchable type="desktop-id">io.github.fabrialberio.pinapp.desktop</launchable>
 	<metadata_license>CC0-1.0</metadata_license>
 	<project_license>GPL-3.0-or-later</project_license>
 	<name translatable='no'>PinApp</name>
@@ -38,6 +38,7 @@
     <content_rating type="oars-1.1"/>
 	<url type="homepage">https://github.com/fabrialberio/PinApp</url>
     <url type="bugtracker">https://github.com/fabrialberio/PinApp/issues</url>
+	<url type="vcs-browser">https://github.com/fabrialberio/PinApp</url>
     <recommends>
 		<control>pointing</control>
 		<control>keyboard</control>
--
libgit2 1.7.2

